tests/test_infer_param_optionality.py

```diff
@@ -0,0 +1,156 @@
+from fastapi import APIRouter, FastAPI
+from starlette.testclient import TestClient
+
+app = FastAPI()  # pylint: disable=invalid-name
```

Comment:
```suggestion
app = FastAPI()
```

We are not using Pylint for linting the internal code, only `mypy`, `black`, and `isort`, so it's safe to remove this comment.